client/src/TreeMap/TreeMapTopbar.tsx

```diff
@@ -13,14 +13,36 @@ const text_maker = create_text_maker([treemap_text]);
 const top_level_title = `${text_maker("government_stats")}`;
 
 const BreadcrumbArrow = (
-  <IconArrow svg_style={{ verticalAlign: "0.1em" }} inline="true" />
+  <IconArrow svg_style={{ verticalAlign: "0.1em" }} inline={true} />
 );
 
-export class TreeMapTopbar extends React.Component {
-  constructor() {
-    super();
+interface TreeMapTopbarProps {
+  history: {
+    action: string;
+    block: Function;
+    createHref: Function;
+    go: Function;
+    goBack: Function;
+    goForward: Function;
+    length: number;
+    listen: Function;
+    location: {
+      hash: string;
+      pathname: string;
+      search: string;
+      state: undefined;
+    };
+    push: Function;
+    replace: Function;
+  };
+  org_route: [];
+  setRouteCallback: Function;
```

Comment:
Looks like this is `(route: string, reset: boolean) => void`. A bit awkward to be setting that in a bunch of places though, could export the interface for `setRoute` from `TreeMap`, that should be shared between this and the other `setRouteCallback`s